plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/ImplementRedshiftLpad.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Could you explain the motivation to support `lpad` function pusdhown in PR description?